frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -225,14 +230,24 @@ class _AppFlowyEditorPageState extends State<AppFlowyEditorPage> {
         CalloutBlockKeys.type
       ];
 
+      final supportAlignBuilderType = [
+        ImageBlockKeys.type,
+      ];
+
       final colorAction = [
         OptionAction.divider,
         OptionAction.color,
       ];
 
+      final alignAction = [
+        OptionAction.divider,
+        OptionAction.align,
+      ];
+
       final List<OptionAction> actions = [
         ...standardActions,
         if (supportColorBuilderTypes.contains(entry.key)) ...colorAction,
+        if (supportAlignBuilderType.contains(entry.key)) ...alignAction,
```

Comment:
Could you make this condition a boolean getter?
```dart
bool get supportsAlignBuilder => supportAlignBuilderType.contains(entry.key);
```

```suggestion
        if (supportsAlignBuilder) ...alignAction,
```

Reply:
I disagree. We cannot use the entry.key directly in the getter, and converting it into a function appears unnecessary.